example/string_generator_test.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python
+"""Test the string generator class."""
+
+import os
+import cherrypy
+from cherrypy.test import helper
+from example.string_generator import StringGeneratorWebService, STRING_FILE
+
+
+class TestStringGenerator(helper.CPWebCase):
+    """The test framework for string generator service."""
+
+    # pylint: disable=no-self-use
+    def setUp(self):
+        """Setup each test by starting the CherryPy server."""
+        self.PORT = 1234
```

Comment:
Okay, but why don't you use it below then? Otherwise, I would suggest moving it outside this method; for example, just make these vars class properties since they don't change